hack/generator/pkg/astmodel/armconversion/convert_from_arm_function_builder.go

```diff
@@ -130,7 +130,9 @@ func (builder *convertFromArmBuilder) assertInputTypeIsArm() []ast.Stmt {
 		result,
 		astbuilder.ReturnIfNotOk(
 			astbuilder.FormatError(
-				fmt.Sprintf("\"unexpected type supplied for FromArm function. Expected %s, got %%T\"", builder.armTypeIdent.Name),
+				fmt.Sprintf("\"unexpected type supplied for %s function. Expected %s, got %%T\"",
```

Comment:
Minor - Add `()` after the function name to make it more obvious that it is the actual function name, and not just a description.
```suggestion
				fmt.Sprintf("\"unexpected type supplied for %s() function. Expected %s, got %%T\"",
```